VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTopic.java

```diff
@@ -80,7 +83,9 @@ public boolean hasSameId(@NonNull DiscussionTopic discussionTopic) {
     }
 
     public boolean containsThread(@NonNull DiscussionThread discussionThread) {
-        if (identifier.equals(discussionThread.getTopicId())) {
+        if (isAllType() || isFollowingType()) {
```

Comment:
Negative. Its isn't logically possible, we can't be on the "Posts I'm following" and "All Posts" screens at once for the condition to be `true`.